homeworks/konstantin.tsarenko_KTsarenko/homework_7/hv_7.js

```diff
@@ -0,0 +1,24 @@
+
+function getRandomInt() {
+    const min = 50;
+    const max = 100;
+    return Math.floor(Math.random() * (max - min)) + min;
+}
+
+function getFibonacciSequence() {
+    const n = getRandomInt();
+    const fibonacci = [0, 1];
+
+    for (let i = 2; ; i++) {
```

Comment:
why not limit your loop with max value you have already randomly generated?

Reply:
sorry, just guided you wrong I guess... `while` loop could be more suitable here